mlrun/api/schemas/client_spec.py

```diff
@@ -55,8 +55,10 @@ class ClientSpec(pydantic.BaseModel):
     function: typing.Optional[Function]
     redis_url: typing.Optional[str]
     redis_type: typing.Optional[str]
+
+    # ce_mode is deprecated, we will use the full ce config instead
     ce_mode: typing.Optional[str]
-    ce_version: typing.Optional[str]
+    ce: typing.Optional[typing.Dict[str, str]]
```

Comment:
This type hinting notes that the `ce` struct will only be of 1 level hierarchy, not sure we want to add that restriction. Take a look how `logs` is being defined and initialized on client side.